python-avd/pyavd/_eos_designs/structured_config/underlay/router_ospf.py

```diff
@@ -31,7 +31,7 @@ def router_ospf(self: AvdStructuredConfigUnderlayProtocol) -> None:
             passive_interface_default=True,
             router_id=self.shared_utils.router_id if not self.inputs.use_router_general_for_router_id else None,
             max_lsa=self.inputs.underlay_ospf_max_lsa,
-            maximum_paths=self.inputs.underlay_ospf_maximum_paths,
+            maximum_paths=self.inputs.underlay_ospf_maximum_paths if self.inputs.underlay_ospf_maximum_paths != 128 else None,
```

Comment:
```suggestion
            # 128 is the default value in EOS and to avoid
            # an extra line in the config we don't render it today
            maximum_paths=self.inputs.underlay_ospf_maximum_paths if self.inputs.underlay_ospf_maximum_paths != 128 else None,
```